bip-0360.mediawiki

```diff
@@ -0,0 +1,617 @@
+<pre>
+  BIP: 360
+  Title: Pay to Quantum Resistant Hash
+  Layer: Consensus (soft fork)
+  Author: Hunter Beast <hunter@surmount.systems>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0360
+  Status: Draft
+  Type: Standards Track
+  Created: 2024-12-18
+  License: BSD-3-Clause
+</pre>
+
+
+== Introduction ==
+
+=== Abstract ===
+
+This document proposes the introduction of a new output type using signatures based on Post-Quantum Cryptography (PQC).
+This approach for adding a post-quantum secure output type does not require a hard fork or block size increase.
+
+=== Copyright ===
+
+This document is licensed under the 3-clause BSD license.
+
+=== Motivation ===
+
+The primary threat to Bitcoin from Cryptoanalytically-Relevant Quantum Computers (CRQCs)<ref name="CRQC">
+A Cryptoanalytically-Relevant Quantum Computer is an ''object'' which is only loosely defined by ''characteristics'' in quantum physics as of today. It could be understood in the context of this BIP and in bitcoin that it's a ''hardware-agnostic'' computer supposed to have the architecture to keep ''coherent'' a sufficient number of logical qubits to be able to run the Shor algorithm in an efficient fashion.</ref>
+is their potential to break the cryptographic assumptions of Elliptic Curve Cryptography (ECC), which secures Bitcoin's signatures and Taproot
+commitments. Specifically, [https://arxiv.org/pdf/quant-ph/0301141 Shor's algorithm] enables a CRQC to solve the
+Discrete Logarithm Problem (DLP) exponentially faster than classical methods<ref name="shor">Shor's algorithm is
+believed to need 10^8 operations to break a 256-bit elliptic curve public key.</ref>, allowing the derivation of
+private keys from public keys—a process referred to as quantum key decryption. Importantly, simply doubling the public
```

Comment:
nit: I tried to look up "quantum key decryption" because it struck me as a very confusing term, but I didn't find any references.

Reply:
I'll change it to, "a process referred to **here**"